assets/js/components/publisher-wins/publishing-win.js

```diff
@@ -50,25 +50,32 @@ const publishingWin = ( data, id ) => {
 			averageCTR,
 		} = processedData;
 
-		dataBlocks = [
-			{
-				title: __( 'Total Impressions', 'google-site-kit' ),
-				datapoint: totalImpressions,
-				datapointUnit: '',
-			},
-			{
-				title: __( 'Total Clicks', 'google-site-kit' ),
-				datapoint: totalClicks,
-				datapointUnit: '',
-			},
-			{
-				title: __( 'Average CTR', 'google-site-kit' ),
-				datapoint: averageCTR,
-				datapointUnit: '%',
-			},
-		];
+		// Only display stats if they are relevant. If everything is 0, it's likely
+		// because the Search Console property is entirely new and hasn't aggregated
+		// any data yet for the site.
+		if ( 0 !== parseInt( totalClicks ) ||
+			0 !== parseInt( totalImpressions ) ||
+			0 !== parseInt( averageCTR ) ) {
```

Comment:
`parseInt` doesn't default to base 10, so we should always call it with `10` as the second parameter to avoid unintended results.
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Description